Misc/NEWS.d/next/Library/2020-09-13-02-02-18.bpo-39337.L3NXTt.rst

```diff
@@ -0,0 +1 @@
+:func:`encodings.normalize_encoding` now ignores non-ASCII letters.
```

Comment:
Hum, since the encodings module has no documentation, it would help to warn users that the behavior changed at least in the What's New in Python 3.10 document. Can you please add an "encodings" section at https://docs.python.org/dev/whatsnew/3.10.html#improved-modules and copy/paste this sentence there? You can add the "(Contributed by Hai Shi in bpo-39337.)" credit ;-)

Reply:
done, updated.